### DISCUSSION TITLE
b站似乎有概率会在“接下来播放”的上方加一个“创作推广”的视频

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3269] [综合讨论 / General]
如题，或许可以添加一个屏蔽它的功能？（如果还没有这个功能的话）

**Comment by Tinhone** [comment]
有图了
![图片](https://user-images.githubusercontent.com/81268244/164753152-58268827-e531-44a4-a898-b4c796d11ed6.png)

**Comment by theblueandblackcrow** [comment]
结合外部的影响
感觉出现这个并不意外
不过这个简单，把它当作广告移除就行了

**Comment by the1812** [comment]
之前就发现了, 并且已经加入 `删除广告` 功能, 下次更新就会有. (0459211)
